homeTask_3/homeTask_3/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace homeTask_3
+{
+    class Program
+    {
+        //Задание- создать три массива по 10 элементов, первый
+        //заполнить значениями индексов элементов, второй - квадратами индексов,
+        //третий-кубами, и вывести на экран средние значения элементов для всех массивов
+
+        public void InitializeArray(int[] array, int n)
+        {
+            for (int i = 0; i < array.Length; i++)
+                array[i] = (int)Math.Pow(i+1, n);
```

Comment:
It's not necessary to cast to int. Because array type int and after variable will be int after pow.

Reply:
@EgorAntonovich  why it is not necessary. The Pow function return double type
![image](https://user-images.githubusercontent.com/99362623/156077037-c3564730-39ec-4c14-bf74-0dfdb9dea0d1.png)
and I need to convert double to int.